web-console/src/views/segments-view.tsx

```diff
@@ -155,15 +178,17 @@ export class SegmentsView extends React.Component<SegmentsViewProps, SegmentsVie
         {
           Header: "Segment ID",
           accessor: "segment_id",
-          width: 300
+          width: 300,
+          show: tableColumnSelection["Segment ID"]
```

Comment:
It looks like if a column is added in a future PR it will be default not shown. You should flip the entire script and only store the hidden columns and show columns unless they are hidden.